### PR TITLE
correctional_facility_type prison type filter added

### PR DESCRIPTION
**Pull request by ilodidoj**
[Link to issue.](https://github.com/usdoj-crt/crt-portal-management/issues/986)

## What does this change?
Allow filter view all table by prison type

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [ ] Re-check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/forms.py [hunk 1]
```diff
@@ -1064,7 +1071,8 @@ class Meta:
             'intake_format',
             'contact_email',
             'referred',
-            'language'
+            'language',
+            'correctional_facility_type',
```
Comment (billyhunt) [inline crt_portal/cts_forms/forms.py:1075]: Do we prefer commas on the last item of an array or not?

**Review by billyhunt — APPROVED** [review]
Looks good to me!

**Review by billyhunt — CHANGES_REQUESTED** [review]
Actually, I am not sure it is working.  When I create a new item with a private prison, and search for it, I see all records, and the selection is not saved on reload.

**Comment by ilodidoj** [comment]
> Actually, I am not sure it is working. When I create a new item with a private prison, and search for it, I see all records, and the selection is not saved on reload.

Hi @billyhunt I am thinking there might be an issue of JS script not updated on your local.  Please close docker, remove all images except the database image from docker images.  Remove the local stack volume only. Rebuild with docker-compose up --build when this PR is checked out.  This should resolve any local stack issue and JS update.  Let me know if you still see the same behavior after doing these steps. Thanks.

**Review by billyhunt — APPROVED** [review]
You were correct Imran.  Wiping everything and starting with new docker containers was my problem.  

All tests pass.  Flake8 tests pass.  The code looks good to me!

**Pull request merged**